routes/order.js

```diff
@@ -666,6 +669,45 @@ router.post(
               where: { bid_logid: req.body.bid_logid },
             }
           )
+            .then(
+              //FOR TESTING EMAILS
+              customSendEmail(
+                'daniela@foodprintlabs.com',
+                'You Accepted a Bid',
+                acceptOrderEmail(
```

Comment:
@DanielmAaron can you create an `orderEmailTemplate.js` module in `config/email` folder which you then import in `order.js` as `let acceptOrderEmail = require('../config/email/email');`

That way, any email templates, even if hardcoded/not pretty for now will be in a central place.